app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -72,11 +72,31 @@ def healthcheck
     end
 
     def facilities
-      render(json: lighthouse_facilities_service.get_facilities(lighthouse_facilities_params))
+      lighthouse_facilities = lighthouse_facilities_service.get_facilities(lighthouse_facilities_params)
+
+      if params[:include_all]
+        render(json: lighthouse_facilities)
+      else
+        active_facilities = ves_active_facilities(lighthouse_facilities)
+
+        render(json: active_facilities)
+      end
     end
 
     private
 
+    def ves_active_facilities(lighthouse_facilities)
+      active_ids = StdInstitutionFacility.active.pluck(:station_number).compact
+
+      lighthouse_facilities.select { |facility| active_ids.include?(station_number(facility)) }
+    end
+
+    def station_number(facility)
```

Comment:
I like how this turned it here. It makes it easier to understand what is going on 😃

Reply:
naming things!